app/controllers/posts_controller.rb

```diff
@@ -30,6 +30,25 @@ def all
     @posts = Post.all.order("created_at DESC")
   end
 
+  # def destroy
+  #   return @error = true if @post.user_id != session[:user_id]
```

Comment:
Should remove all the commented out bits before merging.

Reply:
Seconded